new-client/src/components/Window.js

```diff
@@ -175,17 +178,9 @@ class Window extends React.PureComponent {
         this.updatePosition();
       }
     });
-  }
 
-  componentDidUpdate = (prevProps, prevState) => {
-    if (this.props.open) {
-      //This is ugly but there is a timing problem further down somewhere (i suppose?).
-      //componentDidUpdate is run before the render is actually fully completed and the DOM is ready
-      setTimeout(() => {
-        this.windowRef.current.focus();
-      }, 200);
-    }
-  };
```

Comment:
I don't understand why this is removed? I might be missing something? The snipped was added just 3 weeks ago so that hajk could comply with WCAG. Maybey you've solved it some other way?

Reply:
My bad. I will update.

Reply:
@sweco-sesqqq This is interresting as the settimeout seams to be related to the Android bug #715.

Reply:
What a coincidence! (Or maybey you've fixed #715 for Västtrafik Martin?)

I guess the android bug is more substantial than whatever this snippet fixes. Maybey @sweco-seqtob or @jacobwod got some input on this?

Reply:
This fix is needed to make window focused when opened (implemented due to WCAG requirements). 
It can be removed as long as we can achieve the same functionality in another way?

Problem is that we don't have a callback for when the windows are actually done being painted in the browser (only the postrender method componentDidUpdate). Been trying to use requestAnimationFrame and other functions to make it work.

There shouldn't be any more code running when the setTimeoutCallback is coming back so it would not take more than 200ms and the user wouldn't be able to focus a field in that time, but at the same time it sounds like this is the problem :) (#715)?

Dirty fix  might be to check if it is documenthandler that is opened?

Feel free to refactor if anyone can think of something better :)

Reply:
I propose the following:
1. Martin merges the current `develop` into his branch and makes the PR
2. After the merge, `develop` will still contain Tobias fix for window focus. 
3. Someone, perhaps @jesade-vbg (as you seem to have a test rig ready with an Android phone) makes a local copy of `develop` where he _removes_ the code in `Window.js`, `componentDidUpdate()`, and tests again to see if it changes anything.

Depending on the results of 3. we will either a) remove the fix in `Window.js` (and look for another solution to Tobias' problem!), or b) leave the code as-is (if it doesn't change anything with Jesper's bug) and start looking for a solution to #715 somewhere else.

Reply:
OK, I just saw that @jesade-vbg already did the test and it turns out that the related snippet _is_ causing this bug. 

But let's keep the issues apart: @sweco-semtto merges whatever is in `develop` into his branch, at time of merge, while we investigate and find a solution for #715 (and continue the discussion over there). OK?

Reply:
Great!